images/linux/scripts/installers/yq.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+YQ_VERSION=$(curl -s https://api.github.com/repos/mikefarah/yq/releases/latest | jq -r '.tag_name')
+YQ_BINARY=yq_linux_amd64
+
+# As per https://github.com/mikefarah/yq#wget
+echo "Downloading yq version ${YQ_VERSION}"
+YQ_URL="https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/${YQ_BINARY}.tar.gz"
```

Comment:
We could parse the direct link from: 
`curl -s https://api.github.com/repos/mikefarah/yq/releases/latest | jq -r '.assets[].browser_download_url | select(endswith("yq_linux_amd64.tar.gz"))'`

Reply:
@Yann-J could you please apply the suggested changes?

Reply:
OK I'm not convinced that this is much more readable, but I guess that's fine too